### DISCUSSION TITLE
New  Debugging section in Wiki

### DISCUSSION THREAD

**Discussion by jmalak** [open-watcom/open-watcom-v2#525] [Announcement]
I added new Debugging section into Wiki.
First notes are about debugging DOS application on 64-bit Windows in DOSBOX.
Similar way it can be run on other supported hosts as Linux.
The method use minimal memory overhead on DOS.

**Comment by Abaduaber** [comment]
Thank you very much. I've red that and realized how it is possible to debug 16 bit DOS executable under Win64 using dosbox and serial port debugging.
